### PR TITLE
PN-142: Limit number of matches that appear in the UI

### PR DESCRIPTION
**Pull request by buske**
This is a temporary fix until a "Show more results" solution is implemented.

### CODE REVIEW

**Comment by buske** [comment]
@allasm and @veronikaslc please give this a quick review

**Comment by buske** [comment]
Once this PR is merged, we'll hotfix PC and add the Monarch MME endpoint to the UI.

File: ui/src/main/resources/PhenoTips/SimilarCases.xml [hunk 1]
```diff
@@ -318,6 +319,8 @@
       }
       data.featuresCount = data.query.features ? data.query.features.size() : 0;
 
+      data.resultsCount = Math.min(data.resultsCount, this._MAX_RESULTS_DISPLAYED);
+      data.results = data.results.slice(0, data.resultsCount);
```
Comment (allasm) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: I quickly browsed through the MME code and this file, and don't see where we sort the results by score? They may happen to be sorted by our server (again, MME code does not sort, but I think the underlying findSimilarCases() in patient-network does) - but is there a guarantee remote servers will return results in the sorted order?

Comment (buske) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: Good point. No, there's no guarantee. We should sort the results before paginating.

**Comment by sdumitriu** [comment]
@buske Any news on this?

**Comment by buske** [comment]
@sdumitriu MME servers are not guaranteed to return their results in sorted order. However, all currently do. As a result, we hotfixed PC with this fix and enabled matching with Monarch. This solution is okay for now, but it would be best to sort by score first.

**Pull request merged**